www-plugins/lightspark/lightspark-0.8.1.ebuild

```diff
@@ -12,7 +12,7 @@ SRC_URI="https://github.com/lightspark/lightspark/archive/${PV}.tar.gz -> ${P}.t
 LICENSE="LGPL-3"
 SLOT="0"
 KEYWORDS="~amd64 ~x86"
-IUSE="cpu_flags_x86_sse2 curl ffmpeg gles libav nsplugin ppapi profile rtmp"
+IUSE="cpu_flags_x86_sse2 curl ffmpeg gles-only libav nsplugin ppapi profile rtmp"
```

Comment:
"gles" was a terrible USE flag name, so I'd take this opportunity to change it to "gles2-only" since it controls `-DENABLE_GLES2=...`